src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -373,13 +392,24 @@ void FullyConnected::createPrimitive() {
         return;
     }
 #endif
+
+#ifdef OV_CPU_WITH_LLMDNN
+    if (stateLLMFc == State_Use)
+        return;
+#endif
+
     setPostOps(attr, outDims);
     attr.set_scratchpad_mode(dnnl::scratchpad_mode::user);
     Node::createPrimitive();
     appendPostOpArgs(attr, primArgs, postOpsArgs);
 }
 
 void FullyConnected::prepareParams() {
+#ifdef OV_CPU_WITH_LLMDNN
+    if (tryUseLLMFc())
```

Comment:
If the `tryUseLLMFc` function is to init the kernel and repack the weight, could you move it to `createPrimitive` ? I think this kernel is M-dim agnostic, no need to recreate it during runtime.

Reply:
Currently the pack is done at the very first execution, and needs the activation data, so only inside `prepareParams` the parameters are ready. Will refactor to add a dedicated interface for pack in another PR, ticket created: 116635.

Reply:
A dedicated interface is added for pack, and the logic is moved to `createPrimitive`.